x-pack/plugins/security_solution/public/management/pages/trusted_apps/store/utils.ts

```diff
@@ -5,12 +5,16 @@
  * 2.0.
  */
 
+import { escape } from 'lodash/fp';
+
 export const parseQueryFilterToKQL = (filter: string): string => {
   if (!filter) return '';
   const kuery = [`name`, `description`, `entries.value`, `entries.entries.value`]
     .map(
       (field) =>
-        `exception-list-agnostic.attributes.${field}:*${filter.trim().replace(/\s/gm, '* *')}*`
+        `exception-list-agnostic.attributes.${field}:*${escape(
+          filter.trim().replace(/\s/gm, '*').replace(/\\/gm, '\\\\').replace(/:/gm, '\\:')
```

Comment:
I think you might need to escape the special characters before replacing spaces with `*` - for the case where they might be searching for a path that has wildcards on it. 

Also - should we just escape everything that is not a letter or number just to be safe? there is no danger in doing that, correct?

Reply:
Something like:

```typescript
// Match any non letter and non-number
.replace(/([^a-zA-Z0-9])/gm, '\\$1')
```

Reply:
- I tried some searches with wildcards inside like `C:\*\` and it works.
- Do you mean something like this? `.replace(/\W/gm, '\\$&')`. We still need an special replace for backslashes I think